src/test/java/zmq/io/net/SelectorProviderTest.java

```diff
@@ -0,0 +1,65 @@
+package zmq.io.net;
+
+import java.io.IOException;
+import java.nio.channels.spi.SelectorProvider;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.zeromq.SocketType;
```

Comment:
We could consider splitting the tests into 2 classes, one in zmq namespace, the other one in org.zeromq.